src/main/java/com/waytta/clientinterface/LocalClient.java

```diff
@@ -1,13 +1,60 @@
 package com.waytta.clientinterface;
 
+import hudson.Extension;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+import hudson.util.FormValidation;
+import org.jenkinsci.Symbol;
+
+import com.waytta.Utils;
+
 public class LocalClient extends BasicClient {
+    public static final int DEFAULT_JOB_POLL_TIME = 10;
+    private Integer jobPollTime = DEFAULT_JOB_POLL_TIME;
+    private Boolean blockbuild = false;
```

Comment:
Done. Moved both jobPollTime and blockbuild to `@DataBoudnSetter`. Left the `@DataBoundConstructor` for target and targetType as it is required.